modules/nw-aws-load-balancer-with-outposts.adoc

```diff
@@ -1,11 +1,28 @@
 // Module included in the following assemblies:
 // * networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc
+// * post_installation_configuration/configuring-aws-outposts.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="nw-aws-load-balancer-with-outposts_{context}"]
-= AWS Load Balancer Operator and AWS Outposts
+= Using the AWS Load Balancer Operator with AWS Outposts
 
-AWS Load Balancer Operator can work with AWS Outposts. Application Load Balancer is supported. Network Load balancer is not, so AWS Load Balancer Operator does not provision it on AWS Outposts subnets. Ingress resources must be annotated with the either outposts or regular subnets (but not both).
+You can configure the AWS Load Balancer Operator to provision an AWS Application Load Balancer in an Outpost.
+AWS Outposts does not support AWS Network Load Balancers or Classic Load Balancers.
+As a result, the AWS Load Balancer Operator cannot provision those types of load balancers in an Outpost.
```

Comment:
@alebedev87 is this statement needed? 
"As a result, the AWS Load Balancer Operator cannot provision those types of load balancers in an Outpost."

Per @lihongan, [CLB doesn't need AWS LB Operator](https://github.com/openshift/openshift-docs/pull/71263#pullrequestreview-1892494822), so I'm trying to reconcile these two statements.

Reply:
You can create NLBs (like CCM, in that context they overlap in functionality) with ALBO. So we need to state in the docs that ALBO won't be able to provision NLBs. However CLBs are not used by ALBO, so they can be removed from ALBO's chapter.

Reply:
Aha - that all makes sense now. Thanks!